wiki/Concept-Starting-and-orchestrating-Bots.md

```diff
@@ -22,17 +25,65 @@ I would recommend an infrastructure that would look like the following diagram:
 ![](https://i.imgur.com/OV6onHa.png)
 ![](https://i.imgur.com/dMtsdwm.png)
 
-
 # Orchestrating Bots
+
 I will provide a list with considerations for and against taking care of the orchestration of Bots within our software.
 
 Pro:
-* Allows for more deeply integrated functionality, such as statistics
-* A single user would increase productivity when running multiple Bots at the same time
+
+- Allows for more deeply integrated functionality, such as statistics
+- A single user would increase productivity when running multiple Bots at the same time
 
 Con:
-* Would require additional authentification issues related to running the Bots on the local machine of different users, such as "Who should be allowed to run a Bot on whose computer?"
-* In my opinion this only makes sense **after** the foundation has been laid through the possibility of a single person running multiple Bots at the same time, which as far as my understanding and knowledge goes, would be when the Cloud is available for running the Bots in there
-* Outsourcing the orchestration would allow focusing more on the core functionality and should be fairly easy to do once the APIs have been opened to external services to start Bots
+
+- Would require additional authentification issues related to running the Bots on the local machine of different users, such as "Who should be allowed to run a Bot on whose computer?"
+- In my opinion this only makes sense **after** the foundation has been laid through the possibility of a single person running multiple Bots at the same time, which as far as my understanding and knowledge goes, would be when the Cloud is available for running the Bots in there
+- Outsourcing the orchestration would allow focusing more on the core functionality and should be fairly easy to do once the APIs have been opened to external services to start Bots
 
 My recommendation would be to not take care of the orchestration in-house, but rather let external services take care of that (for now). I can see the benefits, which would especially help the management level through statistics and extended information, but I do not think that the orchestration should be of priority because of the rabbit hole that we might fall into security-wise.
+
+# Communication with ark automate dektop app
+
+I looked at several possible technologies/concepts that be applied in our case.
```

Comment:
Who is I? Please write this in a passive case, like "Multiple concepts could be applied in the case of ark_automate"

Reply:
✅